src/bindings/RescriptCompilerApi.res

```diff
@@ -405,7 +408,13 @@ module Compiler = {
     ConversionResult.decode(~fromLang=Reason, ~toLang=Reason, json)
   }
 
-  @get @scope("ocaml") external ocamlVersion: t => string = "version"
+  @get external ocaml: t => option<{..}> = "ocaml"
```

Comment:
Any reason why you did not type it as `option<Dict.t>` directly? Then you can get rid of the `Obj.magic`.